### PR TITLE
Add workaround for https://youtrack.jetbrains.com/issue/KT-51205

### PR DESCRIPTION
**Pull request by hfhbd**
https://regex101.com/r/hEUuDC/1

### CODE REVIEW

File: kfc/kfc-gradle-plugin/src/main/kotlin/io/github/turansky/kfc/gradle/plugin/DefinitionsPlugin.kt [hunk 1]
```diff
@@ -21,9 +21,13 @@ internal class DefinitionsPlugin : Plugin<Project> {
 
                 if (definitionFile.exists()) {
                     val content = definitionFile.readText()
+                    // https://youtrack.jetbrains.com/issue/KT-51205
+                    val fixKT51205 = """any\/\*\ ([^\*\/]*)\ \*\/""".toRegex()
```
Comment (turansky) [inline kfc/kfc-gradle-plugin/src/main/kotlin/io/github/turansky/kfc/gradle/plugin/DefinitionsPlugin.kt:25]: `:any`?

Comment (turansky) [inline kfc/kfc-gradle-plugin/src/main/kotlin/io/github/turansky/kfc/gradle/plugin/DefinitionsPlugin.kt:25]: `:any`?

**Review by turansky — APPROVED** [review]

File: kfc/kfc-gradle-plugin/src/main/kotlin/io/github/turansky/kfc/gradle/plugin/DefinitionsPlugin.kt (hunk 1 above)
Comment (turansky) [inline kfc/kfc-gradle-plugin/src/main/kotlin/io/github/turansky/kfc/gradle/plugin/DefinitionsPlugin.kt]: `:any`?

**Review by turansky — CHANGES_REQUESTED** [review]

File: kfc/kfc-gradle-plugin/src/main/kotlin/io/github/turansky/kfc/gradle/plugin/DefinitionsPlugin.kt (hunk 1 above)
Comment (hfhbd) [reply kfc/kfc-gradle-plugin/src/main/kotlin/io/github/turansky/kfc/gradle/plugin/DefinitionsPlugin.kt]: Okay, limited it to `: any*/ a */`

File: kfc/kfc-gradle-plugin/src/main/kotlin/io/github/turansky/kfc/gradle/plugin/DefinitionsPlugin.kt [hunk 2]
```diff
@@ -21,9 +21,13 @@ internal class DefinitionsPlugin : Plugin<Project> {
 
                 if (definitionFile.exists()) {
                     val content = definitionFile.readText()
+                    // https://youtrack.jetbrains.com/issue/KT-51205
+                    val fixKT51205 = """: any\/\* ([^\*\/]*) \*\/""".toRegex()
                     val newContent = REDUNDANT_PACKAGES
                         .fold(content) { acc, p ->
                             acc.replace("$p.", "")
+                        }.replace(fixKT51205) {
```
Comment (turansky) [inline kfc/kfc-gradle-plugin/src/main/kotlin/io/github/turansky/kfc/gradle/plugin/DefinitionsPlugin.kt:29]: `": $1"`?

**Review by turansky — CHANGES_REQUESTED** [review]

**Pull request merged**